cypress/integration/seededFlows/adminFlows/pages/landingPage.spec.js

```diff
@@ -28,7 +28,7 @@ describe('Set a landing page from the admin portal', () => {
 
   it('should set a landing page when no other landing page exists', () => {
     cy.findAllByRole('link', { name: 'Edit' }).first().click();
-    cy.findByRole('checkbox', { name: "Use as 'Locked Screen'" }).check();
+    cy.findByTestId('private-checkbox').check();
```

Comment:
Can we change this back to a findByRole? If the label text has to stay the long string, it could just be

```js
cy.findByRole('checkbox', "Use as 'Locked Screen' Determines if this page will be used as a landing page for anonymous viewers.")
```

It's better to test against the roles/text surfaced to users rather than test IDs wherever possible

Reply:
Sure, adding this as todo for Monday when Im back.